whitelist.py

```diff
@@ -0,0 +1,39 @@
+# Because vulture only performs static analysis, this file is used
+# by vulture as a whitelist module and passed as a parameter when
+# running vulture so that it'll know that certain methods are being
+# used.
+
```

Comment:
Is a `whitelist.py` the only way to do this, or can we somehow store it in `setup.cfg` or something?

Tangent: I'm so excited for https://www.python.org/dev/peps/pep-0518/ to become a thing so we can just put all our config in `pyproject.toml`